server/routes/actions/actions_routes.py

```
@@ -414,6 +415,21 @@ def trigger_action(user_id, action_id):
     if body.get("trigger_label"):
         trigger_context["trigger_label"] = body["trigger_label"]
 
+    # on_incident actions (like postmortem) require an incident_id
+    try:
+        with db_pool.get_connection() as conn:
+            with conn.cursor() as cur:
```

Ruff (0.15.13)

[warning] Use a single `with` statement with multiple contexts instead of nested `with` statements

(SIM117)